client/dashboard/data/index.ts

```diff
@@ -132,23 +131,6 @@ export const fetchDomains = async (): Promise< Domain[] > => {
 		.domains;
 };
 
-export const fetchSiteDomains = async ( id: string ): Promise< { domains: SiteDomain[] } > => {
-	try {
-		const domains = await wpcom.req.get( { path: `/sites/${ id }/domains`, apiVersion: '1.2' } );
-		return domains;
-	} catch ( error ) {
-		// TODO: check how to properly fetch for all sites..
-		return { domains: [] };
-	}
-};
-
-export const fetchSitePrimaryDomain = async (
-	siteIdOrSlug: string
-): Promise< SiteDomain | undefined > => {
-	const { domains } = await fetchSiteDomains( siteIdOrSlug );
-	return domains.find( ( domain: SiteDomain ) => domain.primary_domain );
```

Comment:
How did we end up with this "derived fetcher" in the first place?

Reply:
Idk, this is very old, from before the first PR